diff -up envytools-20200304/demmt/mmt_bin2dedma.c~ envytools-20200304/demmt/mmt_bin2dedma.c
--- envytools-20200304/demmt/mmt_bin2dedma.c~	2020-01-08 22:37:16.000000000 +0100
+++ envytools-20200304/demmt/mmt_bin2dedma.c	2020-03-04 23:49:33.170318019 +0100
@@ -37,6 +37,10 @@
 static const int interpret_args_as_text = 0;
 static const int dump_addr1_as_diff = 0;
 
+static struct mmt_txt_nvidia_state
+{
+} mmt_txt_nv_state;
+
 void txt_memread(struct mmt_read *w, void *state)
 {
 	unsigned char *data = &w->data[0];
diff -up envytools-20200304/demmt/mmt_bin2dedma.h~ envytools-20200304/demmt/mmt_bin2dedma.h
--- envytools-20200304/demmt/mmt_bin2dedma.h~	2020-01-08 22:37:16.000000000 +0100
+++ envytools-20200304/demmt/mmt_bin2dedma.h	2020-03-04 23:49:35.538339064 +0100
@@ -19,10 +19,6 @@ void txt_msg(uint8_t *data, unsigned int
 void txt_write_syscall(struct mmt_write_syscall *o, void *state);
 void txt_dup(struct mmt_dup_syscall *o, void *state);
 
-struct mmt_txt_nvidia_state
-{
-} mmt_txt_nv_state;
-
 extern const struct mmt_nvidia_decode_funcs txt_nvidia_funcs;
 extern const struct mmt_nvidia_decode_funcs txt_nvidia_funcs_empty;
 void dump_args(struct mmt_memory_dump *args, int argc);
diff -up envytools-20200304/include/var.h~ envytools-20200304/include/var.h
--- envytools-20200304/include/var.h~	2020-01-08 22:37:16.000000000 +0100
+++ envytools-20200304/include/var.h	2020-03-05 00:01:39.768175014 +0100
@@ -93,13 +93,13 @@ struct vardata {
 	int validated;
 };
 
-enum {
+enum vardata_symtype {
 	VARDATA_ST_FEATURE,
 	VARDATA_ST_VARIANT,
 	VARDATA_ST_VARSET,
 	VARDATA_ST_MODE,
 	VARDATA_ST_MODESET,
-} vardata_symtype;
+};
 
 struct vardata *vardata_new(const char *name);
 void vardata_del(struct vardata *data);
